app/homepage/styles.css

```diff
@@ -14,15 +15,36 @@ body {
   color: white;
 }
 
+ul {
+  padding: 20px;
+  margin: 0;
+}
+li {
+  display: inline;
+}
+
+li a {
+  padding: 10px;
+  text-decoration: none;
+}
+
+li a:hover {
+  padding: 10px;
+  text-decoration: underline;
+}
+
+.active {
+  color: var(--main-color-one);
+  background-color: var(--main-color-two);
+}
```

Comment:
Given that all of these are likely to be used for only the navigation bar, perhaps we should be more specific with our CSS selectors, or else in the future any future usages of `<li>` will be met with confusion

Reply:
And also the same for the other pages